shared/js/content-scope/fingerprint.js

```diff
@@ -0,0 +1,111 @@
+/* global sjcl */
```

Comment:
Is it possible to not leave this global in the document content scope? This will be
 1. Visible to other scripts on the page - could be used for extension detection
 2. Modifiable by other scripts - they could manipulate the api to break randomisation.

Reply:
The content-scope code is bundled together and thrown in an IFFE for this reason. The 'global' here is just external to the file in question but scoped only to the IFFE.

Perhaps having a test to verify this is always the case would be worthwhile though.

Reply:
Added a test.